test/integration/server.js

```diff
@@ -1377,6 +1377,84 @@ describe('Wallet service', function() {
     });
   });
 
+  describe('#getAddressFromWallet', function() {
+    describe('1-of-1 (BIP44 & P2PKH)', function() {
+      var server, wallet;
+      beforeEach(function(done) {
+        helpers.createAndJoinWallet(1, 1, function(s, w) {
+          server = s;
+          wallet = w;
+          w.copayers[0].id.should.equal(TestData.copayers[0].id44);
+          done();
+        });
+      });
+
+      it('should get an address from wallet', function(done) {
+        server.createAddress({}, function(err, address) {
+          should.not.exist(err);
+          should.exist(address);
+          address.walletId.should.equal(wallet.id);
```

Comment:
All these checks are not really needed as `createAddress()` is already tested somewhere else.